packages/docs/src/routes/docs/(qwik)/getting-started/index.mdx

```diff
@@ -68,7 +69,7 @@ yarn create qwik
 bun create qwik@latest
 ```
 
-The CLI guides you through an interactive menu to set the project-name, select one of the starters and asks if you want to install the dependencies. Find out more about the files generated by referring to the [Project Structure](/docs/project-structure/) documentation.
+The CLI guides you through an interactive menu to set the project name, select one of the starters, and ask if you want to install the dependencies.  To find out more about the files generated, refer to the [Project Structure](/docs/project-structure/) documentation.
```

Comment:
```suggestion
The CLI guides you through an interactive menu to set the project-name, select one of the starters, and ask if you want to install the dependencies.  To find out more about the files generated, refer to the [Project Structure](/docs/project-structure/) documentation.
```

project-name made sense because it's suggesting that it's a folder name